src/lightning_app/utilities/proxies.py

```diff
@@ -172,20 +179,15 @@ def sanitize(obj: Union[Path, Drive]) -> Union[Path, Dict]:
 
 class WorkStateObserver(Thread):
     """This thread runs alongside LightningWork and periodically checks for state changes.
-
     If the state changed from one interval to the next, it will compute the delta and add it to the queue which is
     connected to the Flow. This enables state changes to be captured that are not triggered through a setattr call.
-
```

Comment:
Removing these lines will leave the docstring not rendering correctly in the html pages

Reply:
Done !